.github/workflows/autoupdate.yaml

```diff
@@ -0,0 +1,16 @@
+name: autoupdate
+on:
+  push:
+    branches:
+      - develop
+      - 'releases/**'
+jobs:
+  autoupdate:
+    name: autoupdate
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: docker://chinthakagodawita/autoupdate-action-unstable:v1
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          PR_FILTER: "protected"
```

Comment:
what "protected" means here?

Reply:
just means it will only run on protected branches which is fine in our case